stacks_queues/stack.py

```diff
@@ -11,28 +12,40 @@ def __init__(self):
     def push(self, element):
         """ Adds an element to the top of the Stack.
             Returns None
+            stacks -> last in first out 
         """
-        pass
-
+        self.store.add_last(element)
+        return element
     def pop(self):
         """ Removes an element from the top
             Of the Stack
             Raises a StackEmptyException if
             The Stack is empty.
             returns None
         """
-        pass
+        if not self.store:
+            raise StackEmptyException("List is empty")
+
+        return self.store.remove_last()  
+        # tests fails when it returns None
+        
 
     def empty(self):
         """ Returns True if the Stack is empty
             And False otherwise
         """
-        pass
+        return self.store.empty()
 
     def __str__(self):
         """ Returns the Stack in String form like:
             [3, 4, 7]
             Starting with the top of the Stack and
             ending with the bottom of the Stack.
         """
-        pass
+        values = []
```

Comment:
✨ Yes, this works! You could also take advantage of the `str` method in the `LinkedList` class